src/client/app/actions/snippet.ts

```diff
@@ -65,7 +65,7 @@ export class ImportSuccessAction implements Action {
 export class UpdateInfoAction implements Action {
     readonly type = SnippetActionTypes.UPDATE_INFO;
 
-    constructor(public payload: { name: string, description: string }) { }
+    constructor(public payload: { id: string, name?: string, description?: string, gist?: string, isOwned?: boolean }) { }
```

Comment:
Discussed offline. This has been addressed by the field `gistOwnerId` in the `IGist` interface. The check is done in the actual app.